lib/reporters/html/partials/line.html

```diff
@@ -1,36 +1,11 @@
-{{#if this.miss}}
-    {{#if this.chunks}}
-        <tr class="chunks">
-            <td class="line" data-tooltip>{{@key}}</td>
-            {{> lint this.lintErrors }}
-            <td class="hits" data-tooltip>{{this.percent}}</td>
-            <td class="source">{{#each this.chunks}}{{#if this.miss}}<div class="miss {{this.miss}}" data-tooltip>{{this.source}}</div>{{else}}<div>{{this.source}}</div>{{/if}}{{/each}}</td>
-            {{#if sourcemaps}}
-                <td class="sourcemaps file" data-tooltip>{{this.originalFilename}}</td>
-                <td class="sourcemaps line" data-tooltip>{{this.originalLine}}</td>
-            {{/if}}
-        </tr>
-    {{else}}
-        <tr class="miss">
-            <td class="line" data-tooltip>{{@key}}</td>
-            {{> lint this.lintErrors }}
-            <td class="hits" data-tooltip>{{this.percent}}</td>
-            <td class="source" data-tooltip>{{this.source}}</td>
-            {{#if sourcemaps}}
-                <td class="sourcemaps file" data-tooltip>{{this.originalFilename}}</td>
-                <td class="sourcemaps line" data-tooltip>{{this.originalLine}}</td>
-            {{/if}}
-        </tr>
-    {{/if}}
-{{else}}
-    <tr class="hit">
-        <td class="line" data-tooltip>{{@key}}</td>
-        {{> lint this.lintErrors }}
-        <td class="hits" data-tooltip>{{hits this.hits}}</td>
-        <td class="source">{{this.source}}</td>
-        {{#if sourcemaps}}
-            <td class="sourcemaps file" data-tooltip>{{this.originalFilename}}</td>
-            <td class="sourcemaps line" data-tooltip>{{this.originalLine}}</td>
-        {{/if}}
-    </tr>
-{{/if}}
```

Comment:
This is a condensed version of previous code, to limit code duplication, although I agree it's less readable.
I've added a unic ID on each line to allow reference from generated files.

One possible improvement is to remove `this.` which are optional (handlebars implicitly look into this)